docs/decisions/0002-footer-links.rst

```diff
@@ -0,0 +1,168 @@
+2. Footer Links
+########################
+.. The title should be a short noun phrase. For example, "1. Django IDA" or "9. LDAP for Multitenant Integration"; filename should be lowercase with punctuation removed and spaces replaced by dash
+
+Status
+******
+
+**Draft (=> Accepted)**
+
+.. **Accepted** *2023-05-11*
+
+.. Standard statuses
+    - **Draft** if the decision is newly proposed and in active discussion
+    - **Provisional** if the decision is still preliminary and in experimental phase
+    - **Accepted** *(date)* once it is agreed upon
+    - **Superseded** *(date)* with a reference to its replacement if a later ADR changes or reverses the decision
+
+    If an ADR has Draft status and the PR is under review, you can either use the intended final status (e.g. Provisional, Accepted, etc.), or you can clarify both the current and intended status using something like the following: "Draft (=> Provisional)". Either of these options is especially useful if the merged status is not intended to be Accepted.
+
+Context
+*******
+
+.. This section describes the forces at play, including technological, political, social, and project local. These forces are probably in tension, and should be called out as such. The language in this section is value-neutral. It is simply describing facts.
+
+The footer component is basic - it only has a trademark logo and a language
+selector. It doesn't look similar to the legacy UI footer.
+
+It also doesn't have many customization options. You can customize
+LMS base URL and trade mark logo.
+
+If you want to have more advanced customizations, it's necessary to fork the
+component, modify it, and then install the fork as a package instead of the
+default footer at build time of MFEs.
+
+We want to make the MFE footer to closer resemble the legacy UI footer by
+adding the same footer links as in the legacy UI, and to make it easier to
+style the footer component.
```

Comment:
I think the focus is to be to customise the content right? Styling is already feasible with a branding package.

Reply:
@xitij2000 You're right that the content is the main focus. However, currently there is no consensus on how the component can be styled from the outside, other than tweaking the Paragon variables, which is not flexible enough.